pkg/service/sniffer/runtime/crio.go

```diff
@@ -9,20 +9,20 @@ import (
 type CrioBridge struct {
 }
 
-func NewCrioBridge() CrioBridge {
-	return CrioBridge{}
+func NewCrioBridge() *CrioBridge {
+	return &CrioBridge{}
 }
 
-func (c CrioBridge) NeedsPid() bool {
+func (c *CrioBridge) NeedsPid() bool {
 	return true
 }
 
-func (c CrioBridge) BuildInspectCommand(containerId string) []string {
+func (c *CrioBridge) BuildInspectCommand(containerId string) []string {
 	return []string{"chroot", "/host", "crictl", "inspect",
 		"--output", "json", containerId}
 }
```

Comment:
I'm seeing the following failure which is due to no longer having the `/host/` mount. I'm not sure how good an idea it would be to embed `crictl` inside a container image but it may be a better alternative to mounting the entire root FS (this was discussed in #87). What do you think?

```
ERRO[0022] failed executing command: '[chroot /host crictl inspect --output json f4890978c947630aeb8a02620bc34229c975204dd32a2602f638c68196703071]', exitCode: '125', stdErr: 'chroot: cannot change root directory to '/host': No such file or directory
```

Reply:
FYI, it seems feasible to create a util container image with crictl installed using (https://github.com/kubernetes-sigs/cri-tools#install). If you agree this is a better solution compared to having a `/host` mount, I'm happy to submit a follow-up PR.

Reply:
> FYI, it seems feasible to create a util container image with crictl installed using (kubernetes-sigs/cri-tools#install). If you agree this is a better solution compared to having a /host mount, I'm happy to submit a follow-up PR.

This is a good idea! I still prefer this solution than mounting the host path, thank you for catching this!

Reply:
@namco1992 Here's something else to consider. https://github.com/eldadru/ksniff/pull/91 has a dependency on having the `/host` mount.